Draft of GSOD 2020 application (#1056)
alan-turing-institute/the-turing-way: communications/GSOD-applications/2020-gsod.md

# Google Season of Docs (GSOD) 2020

* Website: https://developers.google.com/season-of-docs
* Location: Distributed/online
* Date: 10-11 March 2020
* Context: Technical writers spend three months working closely with an open source community and learn about open source and new technologies. 
* INCF call for the organisation partners: https://www.incf.org/blog/call-mentors-join-us-gsod-2020
* Previous year’s application: https://github.com/alan-turing-institute/the-turing-way/issues/406

### Title

* The Turing Way project expands into a series of books on data research practices

> 

Reviewer: Whats happening with the formatting here? Is the title that long?

Reviewer: Sorry - I meant to go back and rephrase this - I think the title is supposed to capture the expansion but I don't think it's necessary. Just "The Turing Way, a how to guide to data science" would work I think?

**Description:**

> The Turing Way is an open-source, community-led and collaboratively developed book project on making data research accessible for a wider research community ([Website](https://the-turing-way.netlify.com)). 

Reviewer: ```suggestion
The Turing Way is an open-source, community-led and collaboratively developed book project on making data research accessible for a wider research community ([https://the-turing-way.netlify.com](https://the-turing-way.netlify.com)). 
```

We bring together individuals from diverse fields and expertise to develop practices and learning resources that can make data research comprehensible and easy to understand.
The project is openly developed and all questions, comments, recommendations and discussion are facilitated through an [online GitHub repository](https://github.com/alan-turing-institute/the-turing-way).

Our community members are researchers, engineers, data librarians, industry professionals, and experts in various domains, at all levels of seniority, from all around the world. 

> Since last year, they have co-authored chapters on [research reproducibility](https://the-turing-way.netlify.app/reproducibility/reproducibility.html) by compiling best practices, tools and recommendations used by the scientific communities worldwide.  

Reviewer: ```suggestion
Since the project's launch in April 2019, they have co-authored chapters on [research reproducibility](https://the-turing-way.netlify.app/reproducibility/reproducibility.html) by compiling best practices, tools and recommendations used by the scientific communities worldwide.  
```

> Since the project has grown from a small team of 10 researchers to a community of 125 contributors, the scope of The Turing Way book has also expanded to include different aspects such as project design, collaboration, communication, and ethics in data research ([preview](https://deploy-preview-977--the-turing-way.netlify.app/welcome)). The project aims to provide learning and training resources on a wide array of topics and builds on case studies provided by Turing researchers, impact stories contributed by individuals and workshops delivered by the team members. 

Reviewer: ```suggestion
Since the project has grown from a small team of 10 researchers to a community of 125 contributors, the scope of The Turing Way book has also expanded to include different aspects such as project design, collaboration, communication, and ethics in data research ([preview](https://deploy-preview-977--the-turing-way.netlify.app/welcome), pull request [#977](https://github.com/alan-turing-institute/the-turing-way/pull/977)).
The project aims to provide learning and training resources on a wide array of topics and builds on case studies provided by Turing researchers, impact stories contributed by individuals, and workshops delivered by the team members. 
```

> For this project, the GSOD technical writer will review the existing chapters on [research reproducibility](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/contentl). 
> They will also provide editorial support with the chapters on [distributed collaboration](https://github.com/alan-turing-institute/the-turing-way/issues?q=is%3Aissue+is%3Aopen+label%3Acollaboration-book), which are under development. 

Reviewer: ```suggestion
For this project, the GSOD technical writer will review the [existing chapters](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/content) and provide editorial support on incorporating new content, such as the chapters on [distributed collaboration](https://github.com/alan-turing-institute/the-turing-way/issues?q=is%3Aissue+is%3Aopen+label%3Acollaboration-book), which are currently under development. 
```

They will be fairly acknowledged for their contributions to the project. 
The final goal will be to identify opportunities to enhance the quality of the book by improving its language and structure that can make The Turing Way accessible for graduate students, research software engineers, senior investigators and administrative teams.

*Optional technical reading:* 
The Turing Way is hosted as a [Jupyter Book](https://github.com/jupyter/jupyter-book/) at https://the-turing-way.netlify.com. Jupyter Books can format markdown files and Jupyter notebooks as static HTML making them easy to read. 
When a notebook is included in the book, the static page includes a link to an interactive version of the notebook via [Binder](https://mybinder.readthedocs.io). 
We are upgrading to the latest version of Jupyter book and intend to integrate more interactive features, which are currently underutilised. 
The translation efforts are supported through [Transifex](https://www.transifex.com/theturingway/theturingway/dashboard/).

**Required skills:**

- Writing and editing skills in the English language

> - Interest in data or research skills will be useful

Reviewer: ```suggestion
- Experience collaborating on data science or quantitiative research projects at any level
```

> - Know how to work with the distributed community members

Reviewer: ```suggestion
- Experience working in distributed communities, ideally using git and GitHub
```

> - Optional skills in translating English content into Chinese, Hindi or Spanish

Reviewer: ```suggestion

**Optional skills:**

- Translating English content into Chinese, Hindi or Spanish
- Interactive visualisation
- Data science ethics
- Community building
```
